staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/list_work_estimator.go

```diff
@@ -68,7 +68,16 @@ func (e *listWorkEstimator) estimate(r *http.Request, flowSchemaName, priorityLe
 		// return maximumSeats for this request to be consistent.
 		return WorkEstimate{InitialSeats: e.config.MaximumSeats}
 	}
-	isListFromCache := !shouldListFromStorage(query, &listOptions)
+
```

Comment:
I experimented with this PR during my flight, need to double check if it works as expected.

Reply:
Looking at the code it works:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/request/requestinfo.go#L228

and my simple experiments also seemed to work.